js/components/portal-dashboard/feedback/rubric-table.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+// import { Map } from "immutable";
+import Markdown from "markdown-to-jsx";
+import LaunchIcon from "../../../../img/svg-icons/launch-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/rubric-table.less";
+interface IProps {
+  activityStarted: boolean;
+  rubric: any;
+  student: any;
+  rubricFeedback: any;
+  activityId: string;
+  activityIndex: number;
+  updateActivityFeedback?: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
```

Comment:
Does this need to be optional? Don't we always need to pass this in?
` updateActivityFeedback: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;`